controllers/schedule_week.go

```diff
@@ -0,0 +1,400 @@
+package controllers
```

Comment:
Should this be part of the show controller?

Reply:
I don't see why it should be